tests/v2/test_waku_rln_relay.nim

```diff
@@ -639,34 +639,41 @@ suite "Waku rln relay":
     check:
       verified.value() == false
 
-  test "invalidate messages with a valid, but stale root":
+  test "validate roots which are part of the acceptable window":
     # Setup: 
-    # This step consists of creating the rln instance,
+    # This step consists of creating the rln instance and waku-rln-relay,
     # Inserting members, and creating a valid proof with the merkle root
+
+    require:
+      AcceptableRootWindowSize < 10
```

Comment:
This requirement is misleading and might cause confusion when reading the test. Also, this is a condition that we can prevent it if we make sure the Merkle tree is populated with more that the `AcceptableRootWindowSize` keys i.e., line 665.

Reply:
Addressed in [39a759c](https://github.com/status-im/nwaku/pull/1177/commits/39a759c2594cb52f420c9928b268bf65ad70a340)